### PR TITLE
feat: add Qualcomm IoT page

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Add Qualcomm IoT page and contact form.

Copy doc for the webpage: https://docs.google.com/document/d/1BP2T63R6kdnUFQFIZcbXbTIgY-9fQVgVRLsSUU5Rvx8/edit?tab=t.0
Copy doc for the form: https://www.google.com/url?q=https://docs.google.com/document/d/1vZepBdXUWbidF3BD1zwvudiykRjtItHT4ssCqj6Dh-U/edit?tab%3Dt.0&sa=D&source=docs&ust=1731909684091428&usg=AOvVaw3Gjv_xqSPKCnQoYfDiXgwx

## QA

- Open the demo https://ubuntu-com-14500.demos.haus/
- Check that there is a link to the new page:
 -- In the top nav click 'Get Ubuntu'.
 -- To the side click 'Ubuntu for IoT'
 -- Under 'Quick links' there should be a link: 'Install Ubuntu on Qualcomm IoT Platforms'.
- Visit https://ubuntu-com-14500.demos.haus/download/qualcomm-iot
- Click 'Get in touch' and the new form should appear.

## Issue / Card

https://warthogs.atlassian.net/browse/WD-16036

## Screenshots

iot

Screenshot 2024-11-26 at 14-48-26 Install Ubuntu on Qualcomm IoT Platforms Ubuntu

### CODE REVIEW

File: templates/download/qualcomm-iot.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        <p>
+          Canonical partners with silicon vendors, board manufacturers and leading enterprises to shorten time-to-market. If you are deploying Ubuntu on Qualcomm IoT platforms at-scale, <a href="https://ubuntu.com/internet-of-things/qualcomm/contact-us">reach out to Canonical</a> to get access to ongoing bug fixes, critical security patching, long-term support; or to learn more about our solutions for custom board enablement and application development services.
+        </p>
+        <p>
+          For information about using Canonical’s Ubuntu images on a commercial project, <a href="https://ubuntu.com/legal/intellectual-property-policy">read Canonical’s IP policy</a>.
+        </p>
+        <p>
+          If you have any questions about the platform or would like information
+          about our certification program, <a
+          href="/download/contact-us#contact-us" class="p-button--positive
```
Comment (akbarkz) [inline templates/download/qualcomm-iot.html:150]: this should be a simple link, according to design, not a button

File: templates/download/qualcomm-iot.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <a href="https://www.qualcomm.com/">
+          {{ image (
```
[the remark this answers is not in the document]
Comment (huwshimi) [reply templates/download/qualcomm-iot.html:26]: Fixed.

File: templates/download/qualcomm-iot.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="row p-strip is-shallow u-no-padding--top">
+          <hr class="col-9 p-rule--muted" />
+          <div class="col-3">
+            <h3 class="p-heading--5">QCS6490</h3>
+          </div>
+          <div class="col-6 u-vertically-center u-align--center p-image-wrapper is-product">
+            {{ image (
```
[the remark this answers is not in the document]
Comment (huwshimi) [reply templates/download/qualcomm-iot.html]: Done.

File: templates/download/qualcomm-iot.html (hunk 1 above)
Comment (huwshimi) [reply templates/download/qualcomm-iot.html]: Oops! Thanks, fixed.

File: templates/download/qualcomm-iot.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            <p>
+              Get started with Ubuntu Server 22.04 for <a href="https://www.qualcomm.com/products/internet-of-things/industrial/building-enterprise/qcs6490#overview">QCS6490</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              Ubuntu Server 22.04 for QCS6490 <a href="https://docs.qualcomm.com/bundle/resource/topics/80-82645-1/getting-started-with-qualcomm-rb3-gen-2-development-kit_1.html?product=1601111740057201#sub$download-images-and-boot-firmware_1_3">image installation instructions</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              Ubuntu Server 22.04 for QCS6490 <a href="TODO">Release Notes (TODO)</a>
```
Comment (huwshimi) [inline templates/download/qualcomm-iot.html:115]: The release notes link needs to be added here.

**Comment by mattea-turic** [comment]
Thanks @huwshimi ! Just a couple comments:

- Could you add shallow padding below "Ubuntu Server 22.04 for QSCS6490 Release Notes" that's in the section "QCS6490 Qualcomm RB3 Gen 2 Vision kit" under "get started"?
Screenshot 2024-11-26 at 09 18 12

- There also looks to be too much bottom padding for that section too (looks like 3*24px) – could this just be regular pls/
- Could you add a `</br>` after "a" in the section "Are you working on a commercial project?"
- There should also be deep bottom padding
Screenshot 2024-11-26 at 09 22 06

**Comment by eliman11** [comment]
Thanks @huwshimi for your work on this! A few comments:
- [ ] [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
- [ ]  Alt text currently missing for the image, also posted a question out to stakeholders.
- [ ]  Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
Screenshot 2024-11-26 at 13 51 48

**Comment by eliman11** [comment]
> Thanks @huwshimi for your work on this! A few comments:
> 
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.
> * [ ]   Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
> 
> Screenshot 2024-11-26 at 13 51 48

**Comment by eliman11** [comment]
Given the urgency of this task please feel free to add UX+1 if you're just waiting for me and I don't hear back re the link and alt text. We can fix these later so I'm not a blocker!

**Comment by mattea-turic** [comment]
> Thanks @huwshimi for your work on this! A few comments:
> 
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.
> * [ ]   Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
> 
> Screenshot 2024-11-26 at 13 51 48

Button is correct, this is how it wraps in vanilla ([example](https://vanillaframework.io/docs/examples/base/button?theme=light))
Screenshot 2024-11-26 at 15 49 11

**Comment by eliman11** [comment]
Thanks for the clarification @mattea-turic! You can ignore my last bullet point in that case. 

- [ ]  One additional quick fix - would you be able to change the field type for the phone number to the one we have with a preset drop-down for country codes? [Example here](https://canonical.com/#get-in-touch)
Screenshot 2024-11-26 at 13 36 03

**Comment by huwshimi** [comment]
> Thanks @huwshimi ! Just a couple comments:
> 
> * Could you add shallow padding below "Ubuntu Server 22.04 for QSCS6490 Release Notes" that's in the section "QCS6490 Qualcomm RB3 Gen 2 Vision kit" under "get started"?
> * There also looks to be too much bottom padding for that section too (looks like 3*24px) – could this just be regular pls/

These two should be fixed:

Screenshot 2024-11-27 at 2 57 44 pm



> * Could you add a `</br>` after "a" in the section "Are you working on a commercial project?"

Done. I've adjusted this as the screen gets smaller otherwise the title breaks like this:

Screenshot 2024-11-27 at 3 00 41 pm

Screenshot 2024-11-27 at 3 01 29 pm

So I've adjusted the smaller screens to work just break the title naturally:

Screenshot 2024-11-27 at 3 01 01 pm

Screenshot 2024-11-27 at 3 01 16 pm

Is that OK?

> * There should also be deep bottom padding

Done:

Screenshot 2024-11-27 at 3 17 06 pm

**Comment by huwshimi** [comment]
> Thanks for the clarification @mattea-turic! You can ignore my last bullet point in that case.
> 
> * [ ]   One additional quick fix - would you be able to change the field type for the phone number to the one we have with a preset drop-down for country codes? [Example here](https://canonical.com/#get-in-touch)
> 

It looks like this feature isn't currently working for modal forms, I've filed an issue here: https://github.com/canonical/ubuntu.com/issues/14509.

**Comment by eliman11** [comment]
Noted on the forms, thank you for escalating! Received a reply from the stakeholder re the following
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update

Could you please change the link here to the same [form in the hero CTA](https://ubuntu-com-14500.demos.haus/download/qualcomm-iot#get-in-touch) 

> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.

Please add the alt text 'Qualcomm® RB3 Gen 2 Development Kit'. Also marked this in the copydoc.

**Comment by huwshimi** [comment]
> Noted on the forms, thank you for escalating! Received a reply from the stakeholder re the following
> 
> > * [ ]   [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> 
> Could you please change the link here to the same [form in the hero CTA](https://ubuntu-com-14500.demos.haus/download/qualcomm-iot#get-in-touch)

Done.

> 
> > * [ ]    Alt text currently missing for the image, also posted a question out to stakeholders.
> 
> Please add the alt text 'Qualcomm® RB3 Gen 2 Development Kit'. Also marked this in the copydoc.

Done.

**Review by akbarkz — APPROVED** [review]

**Pull request merged**